_schema.graphql

```diff
@@ -1,3 +1,5 @@
+directive @canonicalResource on FIELD
```

Comment:
Can we have documentation for directives?

Reply:
@mzikherman bump

Reply:
double-checking...